roles/database/tasks/main.yml

```diff
@@ -10,3 +10,7 @@
       - name: "{{ database_user }}"
         priv: "{{ database_name }}.*:ALL"
         password: "{{ database_password }}"
+      - name: "{{ database_metabase_user }}"
+        priv: "*.*:REQUIRESSL/{{ database_name }}.people:SELECT(id, uuid, created_at, updated_at, is_admin, locale, preferences, active_days_count, last_page_load_date, test_group_number, username, reset_password_sent_at, remember_created_at, sign_in_count, current_sign_in_at, last_sign_in_at, given_name, family_name, display_name, phone_number, description, image_updated_at)/{{ database_name }}.categories:SELECT/{{ database_name }}.comments:SELECT/{{ database_name }}.conversations:SELECT/{{ database_name }}.follower_relationships:SELECT/{{ database_name }}.listings:SELECT/{{ database_name }}.locations:SELECT/{{ database_name }}.messages:SELECT/{{ database_name }}.participations:SELECT/{{ database_name }}.shipping_addresses:SELECT/{{ database_name }}.stripe_payments:SELECT/{{ database_name }}.testimonials:SELECT/{{ database_name }}.transactions:SELECT/{{ database_name }}.transaction_processes:SELECT/{{ database_name }}.transaction_transitions:SELECT"
```

Comment:
could we split this using something like `loop_var` or even join a list var of some sort to make it readable? I cannot even review it :cry:

Reply:
I believe something like the following could work and would make it more changeable

```
- name: "{{ database_metabase_user }}"
  priv: "{{ privileges | join(':') }}"
```

we could then, define the `privileges` list in `inventory/group_vars/`.

Reply:
It was a POC, if it doesn't cost much time we can try a way to split it.
I would do that on a next iteration when we'll need to change it.